Aptoide/aptoide-client-v8
# [bugfix/ASV-859] similar and in app message layout after install

**What does this PR do?**

  It removes the similar apps header if there's no similar apps
  It removes the apps messages when the is downloading
  It puts the similar apps layout below the appc message layout

**Database changed?**

   No

**Where should the reviewer start?**

NewAppViewFragment.java

**How should this be manually tested?**

Go to an app with in-app purchase with appcoins (2048) press install and check if the appc message disappears.
Comment the code of the mapToSimilar in NewAppViewFragment so that it returns an empty list. Go to an app with in-app purchase with appcoins (2048), scroll down to check if there's the header of the similar apps (there shouldn't be one). press install and check if similar apps appear (it shouldn't).
**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-859

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
